vale/vocab.txt

```diff
@@ -0,0 +1,126 @@
+addr
+api
+architecting
+Artifactory
+artifactory
+Atlassian
+autogenerated
+Autoscaling
+autoscaling
+Basecamp
+Basscss
+Bazel
+Bitium
+boolean
+Boomper
+boomper
+Buildbox
+Buildkit
+Buildkite
+buildkite
+buildscript
+chatbot
+chroot
+chroots
+cli
+composable
+config
+cpu
+cron
+Crontab
+crontab
+Datadog
+Dockerfile
+dogstatsd
+env
+eval
+Explainshell
+fff
+ffxdq
+Firewalled
+Flexbox
+gcloud
+gif
+GitHub
+GitLab
+globbing
+golang
+Gradle
+gz
+Hashicorp
+heroku
+hostname
+http
+https
+Inlining
+inlining
+Javascript
+jpegs
+json
+junit
+kubectl
+kustomize
+linux
+Memcache
+Minitest
+mutex
+namespace
+negator
+Noreply
+npm
+nssm
+Okta
+Pagerduty
```

Comment:
We can add OAuth here :)

Reply:
`oAuth` doesn't show up as a misspelling (probably because camel case).

I'll add it a different check.

Reply:
Thanks @JuanitoFatas !